src/FSharp.Formatting.Common/Utils.fs

```diff
@@ -2,7 +2,11 @@ module FSharp.Formatting.Common.Utils
 #if NETSTANDARD_2_1_OR_GREATER
 #else
 type System.String with
-    member x.StartsWith(c: char) = string c |> x.StartsWith
-    member x.EndsWith(c: char) = string c |> x.EndsWith
-    member x.Contains(c: char) = string c |> x.Contains
+    member x.StartsWith c =
+        x.StartsWith(string<char> c, false, System.Globalization.CultureInfo.InvariantCulture)
```

Comment:
https://learn.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#recommendations-for-string-usage

> Don't use string operations based on [StringComparison.InvariantCulture](https://learn.microsoft.com/en-us/dotnet/api/system.stringcomparison#system-stringcomparison-invariantculture) in most cases. One of the few exceptions is when you're persisting linguistically meaningful but culturally agnostic data.

https://g-research.github.io/fsharp-analyzers/analyzers/StringAnalyzer.html

Reply:
thanks, will give it another shot!